### PR TITLE
MAGECLOUD-4736: Document Changes to Magento-Cloud-Patches

### PR DESCRIPTION
**Pull request by billygilbert**
## Purpose of this pull request

This pull request (PR) updates DevDocs to reflect de-composing Magento Cloud Patches from ECE-Tools

## Affected DevDocs pages

- cloud/project/project-patch
- cloud/release-notes/mcp-release-notes

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/project/project-patch.md [hunk 1]
```diff
@@ -1,51 +1,56 @@
...
-title: Apply custom patches
+title: Apply patches
 functional_areas:
   - Cloud
   - Upgrade
 ---
+The [Magento Cloud Patches](https://github.com/magento/magento-cloud-patches) package is a set of patches, previously used within the ece-tools package to improve the integration of all Magento versions with Cloud environments and to deliver critical fixes quickly. This can also be integrated to an existing Magento project as a standalone package.
 
-Sometimes we provide a [custom patch]({{ site.baseurl }}/guides/v2.3/comp-mgr/patching.html#custom-patches) to address a specific issue. Also, third-party extension developers can provide a custom patch. Copy the custom patch to the `/m2-hotfixes` directory and test it on your local workstation.
+Magento Cloud Patches can also apply [custom patches]({{ site.baseurl }}/guides/v2.3/comp-mgr/patching.html#custom-patches) provided by support or third-party extension developers. Copy the custom patch to a directory called `/m2-hotfixes` in the project root and test it on your local workstation.
```
Comment (meker12) [inline src/cloud/project/project-patch.md]:
```suggestion
Magento Cloud Patches can also apply [custom patches]({{ site.baseurl }}/guides/v2.3/comp-mgr/patching.html#custom-patches) provided by support or third-party extension developers. Copy the custom patch to a directory called `/m2-hotfixes` in the project root, and then test it on your local workstation.
```

File: src/cloud/project/project-patch.md [hunk 2]
```diff
@@ -1,51 +1,56 @@
...
 
     ```bash
-    git add -A && git commit -m "Apply patch" && git push origin <branch name>
+    php ./bin/magento cache:clean
     ```
 
-1. After test validation, merge this branch with the `integration` branch.
+    You can also clean the cache using the [Magento Admin Cache Management](http://docs.magento.com/m2/ee/user_guide/system/cache-management.html).
+
+1. Test the patches, make any necessary changes to custom patches.
```
Comment (meker12) [inline src/cloud/project/project-patch.md]: So for Magento-provided patches, you can test them in your local environment using these instructions, but if there are any issues with the patches, wouldn't you need to contact Magento?

Not clear on how patches work.  

Case 1:
Are patches automatically applied by ece-tools when you deploy your project?  So if you we release a new version of ece-tools, the customer upgrades, then any new patches would be applied automatically  when they deploy their project, or 

Case 2:
Customer has to add the Cloud patches command to their composer file, and then run the `apply patches` command manually.

File: src/cloud/project/project-patch.md (hunk 1 above)
Comment (billygilbert) [reply src/cloud/project/project-patch.md]: Can't apply this suggestion for some reason.

**Pull request merged**